api/apps/api/src/modules/blm/values/project-blm-repo.ts

```diff
@@ -3,10 +3,14 @@ import { Either } from 'fp-ts/Either';
 export const unknownError = Symbol(`unknown error`);
 export const projectNotFound = Symbol(`project not found`);
 export const alreadyCreated = Symbol(`project already has defaults`);
+export const forbidden = Symbol(`access unauthorized`);
```

Comment:
why would blm-repo need to know anything about ACL? it isn't within its boundaries.

Reply:
in this case, it may belong to `projects`